mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -877,6 +853,13 @@ class WidgetAdapter(
         override fun stop() {
             chart.cancelRefresh()
         }
+
+        override fun onClick(v: View?) {
+            v ?: return
```

Comment:
`val context = v?.context ?: return` (and use that below) would be more elegant, I think.

Reply:
Done